pkg/blobstore/configuration/create_blob_access.go

```diff
@@ -261,12 +261,41 @@ func createBlobAccess(configuration *pb.BlobAccessConfiguration, options *blobAc
 				backend.Redis.ReplicationCount,
 				replicationTimeout)
 		case *pb.RedisBlobAccessConfiguration_Single:
+
+			dialTimeout := 5 * time.Second
+			if mode.Single.DialTimeout != nil {
+				dialTimeout, err = ptypes.Duration(mode.Single.DialTimeout)
+				if err != nil {
+					return nil, err
```

Comment:
Though I know that the rest of this source file is particularly bad at this, could you change the error handling to be a bit like this:
```
return nil, util.StatusWrap(err, "Failed to obtain dial timeout")
```
That way it's more obvious to the user where the error comes from.

Reply:
Absolutely, I will change it later on today,.